Podcast/DiscoverViewController.swift

```diff
@@ -21,85 +21,39 @@ class DiscoverViewController: UIViewController, UITableViewDelegate, UITableView
     /// Mark: Variables
     ///
     var feedTableView: UITableView!
-    var categoryCollectionView: UICollectionView!
-    var feedEpisodes: [Episode] = []
-    var categories: [String] = ["News","Money & Business","Politics","Music"]
-    var trendingButton: UIButton!
-    var trendingTagged: Bool = true
-    var categoriesButton: UIButton!
-    var bottomLineView: UIView!
-    var topView: UIView!
+    var feedEpisodes: [Card] = []
     
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .podcastWhiteDark
-        
-        topView = UIView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: topViewHeight))
-        view.addSubview(topView)
-        
-        //collectionview
-        categoryCollectionView = UICollectionView(frame: CGRect(x: 0, y: topView.frame.height, width: view.frame.width, height: view.frame.height - topView.frame.height), collectionViewLayout: PodcastCollectionViewFlowLayout())
-        categoryCollectionView.delegate = self
-        categoryCollectionView.dataSource = self
-        categoryCollectionView.backgroundColor = .podcastWhiteDark
-        categoryCollectionView.register(CategoriesCollectionViewCell.self, forCellWithReuseIdentifier: "CategoriesCollectionViewIdentifier")
-        view.addSubview(categoryCollectionView)
-        categoryCollectionView.reloadData()
-        
+        navigationController?.navigationBar.titleTextAttributes = UIFont.navigationBarDefaultFontAttributes
+        title = "Feed"
+
         //tableview
-        feedTableView = UITableView(frame: CGRect(x: 0, y: topView.frame.height, width: view.frame.width, height: view.frame.height - topView.frame.height))
+        feedTableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height - TabBarController().tabBarHeight))
```

Comment:
don't initialize a new VC to get tab bar height, use app delegate to access tab bar height, or make that a static variable on the tab bar controller class